src/sentry/api/serializers/models/group.py

```diff
@@ -188,8 +188,11 @@ def serialize(self, obj, attrs, user):
         else:
             status_label = 'unresolved'
 
-        permalink = absolute_uri(reverse('sentry-group', args=[
-            obj.organization.slug, obj.project.slug, obj.id]))
+        if user.is_authenticated() and obj.organization in user.get_orgs():
```

Comment:
After looking more closely, I think this behavior of doing this conditionally inside this serializer will be fine. But I will suggest this logic instead:

```python
if user.is_authenticated() and user.get_orgs().filter(id=obj.organization_id).exists():
  ...
```

The change here is pretty subtle, but is a decent performance improvement.

`User.get_orgs()` returns back lazy Django QuerySet, so the query hasn't been evaluated yet. This means we have the ability to continue filtering the query down before we round trip to the database.

So rather than query all of the rows back, we're able to just ask the database to just match against our one row. Then return back a boolean True/False if the row exists, since we don't need to materialize the row at all.